.github/workflows/release.yml

```diff
@@ -14,14 +24,12 @@ jobs:
           authkey: ${{ secrets.TAILSCALE_API_KEY }}
           version: "1.38.4"
       - uses: actions/checkout@v2
-        with:
-          ref: replay-playwright
-      - uses: actions/checkout@v2
-        with:
-          ref: webreplay-release
```

Comment:
This could be kept but we don't need two checkouts nor do we need to specify the branch since it'll use the workflow's branch by default. Doing so prevented testing this before landing because the action always used `webreplay-release` instead of the PR branch.